src/DoctrineORMModule/Service/DoctrineObjectHydratorFactory.php

```diff
@@ -27,9 +28,16 @@ class DoctrineObjectHydratorFactory implements FactoryInterface
     /**
      * {@inheritDoc}
      */
-    public function createService(ServiceLocatorInterface $serviceLocator)
+    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
     {
-        $parentLocator = $serviceLocator->getServiceLocator();
```

Comment:
nop, I have moved `$serviceLocator->getServiceLocator()` into `createService()` method and SMv3 uses `__invoke()` always and $container allways will be ServiceManager instance not a plugin manager.